dashboards-observability/public/components/application_analytics/components/app_table.tsx

```diff
@@ -27,28 +28,64 @@ import {
 } from '@elastic/eui';
 import _ from 'lodash';
 import React, { ReactElement, useEffect, useState } from 'react';
-import { AppAnalyticsComponentDeps, ApplicationType } from '../home';
+import { AppAnalyticsComponentDeps } from '../home';
+import { getCustomModal } from '../helpers/modal_containers';
 import { pageStyles } from '../../../../common/constants/shared';
+import { ApplicationListType } from 'common/constants/application_analytics';
```

Comment:
not sure if this will work, should it be `../../../../common`?

Reply:
Oh yea, weird that it didn't complain. I'll add it to be safe!